install.sh

```diff
@@ -177,6 +177,23 @@ nvm_check_global_modules() {
   fi
 }
 
+nvm_bash_completion() {
+  if [ -f "$HOME/.bashrc" ] || [ -f "$HOME/.bash_profile" ] || [ -f "$HOME/.zshrc" ] ; then
```

Comment:
Okay, updated, though the checking here won't have any performance issue I think.